File: app/src/main/java/org/stepic/droid/web/NotificationResponse.java
package org.stepic.droid.web;

import org.stepic.droid.notifications.model.Notification;
import org.stepik.android.model.Meta;

import java.util.List;

public class NotificationResponse extends MetaResponseBase {

    List<Notification> notifications;

    public NotificationResponse(Meta meta) {
        super(meta);
    }

    public List<Notification> getNotifications() {
        return notifications;
    }
}

File: app/src/main/java/org/stepik/android/remote/notification/model/NotificationResponse.kt
package org.stepik.android.remote.notification.model

import com.google.gson.annotations.SerializedName
import org.stepic.droid.notifications.model.Notification
import org.stepik.android.model.Meta
import org.stepik.android.remote.base.model.MetaResponse

class NotificationResponse(
    @SerializedName("meta")
    override val meta: Meta,
    @SerializedName("notifications")
    val notifications: List<Notification>
) : MetaResponse